internal/controller/compute/statefulserverset/statefulserverset_test.go

```diff
@@ -453,6 +453,7 @@ func Test_areDataVolumesUpToDate(t *testing.T) {
 	type want = struct {
 		creationUpToDate bool
 		areUpToDate      bool
+		areAvailable     bool
```

Comment:
Please add test scenarios for this new boolean. I tried to modify the value to true and there was not test failures:
![image](https://github.com/ionos-cloud/crossplane-provider-ionoscloud/assets/73175496/76b37bc5-1426-4a48-be45-5f8871005cf0)